internal/storages/s3/session.go

```diff
@@ -6,10 +6,15 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/pkg/errors"
+	"os"
 	"strconv"
 	"strings"
 )
 
+const (
+	S3_CA_CERT_FILE = "WALG_S3_CA_CERT_FILE"
```

Comment:
Add this key to internal/config.go in allowedConfigKeys

Reply:
Do we still use os.Getenv(S3_CA_CERT_FILE) ?

Reply:
Try to use internal.GetSettingValue

Reply:
I initially tried to use `internal.GetSettingValue(S3_CA_CERT_FILE)` which required me to import `github.com/wal-g/wal-g/internal`, that in turn gave me an import cycle error during build:
```
import cycle not allowed
package github.com/wal-g/wal-g/main/pg
        imports github.com/wal-g/wal-g/cmd/pg
        imports github.com/wal-g/wal-g/internal
        imports github.com/wal-g/wal-g/internal/storages/s3
        imports github.com/wal-g/wal-g/internal
Makefile:16: recipe for target 'pg_build' failed
```
So I kept getEnv() for the time being.

Reply:
I initially tried to use `internal.GetSettingValue(S3_CA_CERT_FILE)` which required me to import `github.com/wal-g/wal-g/internal`, that in turn gave me an import cycle error during build:
```
import cycle not allowed
package github.com/wal-g/wal-g/main/pg
        imports github.com/wal-g/wal-g/cmd/pg
        imports github.com/wal-g/wal-g/internal
        imports github.com/wal-g/wal-g/internal/storages/s3
        imports github.com/wal-g/wal-g/internal
Makefile:16: recipe for target 'pg_build' failed
```
Basically internal/storage_adapter.go imports internal/storages/s3, and now I have to import internal in internal/storages/s3, which become a circular thing.
So I kept getEnv() for the time being.

Reply:
I found a better way: getting values from settingNames.